sql/telemetry_derived/error_aggregates/query.sql

```diff
@@ -0,0 +1,185 @@
+    FROM
+      UNNEST(SPLIT(REPLACE(TRIM(input, '{}'), '"', ''), ',')) AS entry
+    WHERE
+      LENGTH(entry) > 0 ));
+CREATE TEMP FUNCTION
+  udf_json_extract_histogram (input STRING) AS (STRUCT(
+    CAST(JSON_EXTRACT_SCALAR(input, '$.bucket_count') AS INT64) AS bucket_count,
+    CAST(JSON_EXTRACT_SCALAR(input, '$.histogram_type') AS INT64) AS histogram_type,
+    CAST(JSON_EXTRACT_SCALAR(input, '$.sum') AS INT64) AS `sum`,
+    ARRAY(
+      SELECT
+        CAST(bound AS INT64)
+      FROM
+        UNNEST(SPLIT(TRIM(JSON_EXTRACT(input, '$.range'), '[]'), ',')) AS bound) AS `range`,
+    udf_json_extract_int_map(JSON_EXTRACT(input, '$.values')) AS `values` ));
+CREATE TEMP FUNCTION udf_histogram_get_sum(histogram_list ANY TYPE, target_key STRING) AS (
+  (
+    SELECT
+      udf_json_extract_histogram(value).sum
+    FROM
+      UNNEST(histogram_list)
+    WHERE
+      key = target_key
+    LIMIT
+      1
+  )
+);
+CREATE TEMP FUNCTION udf_round_timestamp_to_minute(timestamp_expression TIMESTAMP, minute INT64) AS (
+  TIMESTAMP_SECONDS(
+    CAST((FLOOR(UNIX_SECONDS(timestamp_expression) / (minute * 60)) * minute * 60) AS INT64)
+  )
+);
+--
+WITH crash_ping_agg AS (
+  SELECT
+    submission_timestamp,
+    normalized_channel AS channel,
+    environment.build.version,
+    application.display_version,
+    environment.build.build_id,
+    metadata.uri.app_name,
+    IF(metadata.uri.app_name = 'Fennec'
+       AND environment.system.os.name = 'Linux',
+       'Android',
+       environment.system.os.name) AS os,
+    environment.system.os.version AS os_version,
+    environment.build.architecture,
+    normalized_country_code AS country,
+    IF(
+      payload.process_type = 'main'
+      OR payload.process_type IS NULL,
+      1,
+      0
+    ) AS main_crash,
+    IF(
+      REGEXP_CONTAINS(payload.process_type, 'content'),
+      1,
+      0
+    ) AS content_crash,
+    IF(payload.metadata.startup_crash = '1', 1, 0) AS startup_crash,
+    IF(
+      REGEXP_CONTAINS(payload.metadata.ipc_channel_error, 'ShutDownKill'),
+      1,
+      0
+    ) AS content_shutdown_crash,
+    -- 0 columns to match main pings
+    0 AS usage_hours,
+    0 AS gpu_crashes,
+    0 AS plugin_crashes,
+    0 AS gmplugin_crashes
+  FROM
+    `moz-fx-data-shared-prod.telemetry_live.crash_v4`
+),
+main_ping_agg AS (
+  SELECT
+    submission_timestamp,
+    normalized_channel AS channel,
+    environment.build.version,
+    application.display_version,
+    environment.build.build_id,
+    metadata.uri.app_name,
+    environment.system.os.name AS os,
+    environment.system.os.version AS os_version,
+    environment.build.architecture,
+    normalized_country_code AS country,
+    -- 0 columns to match crash ping
+    0 AS main_crash,
+    0 AS content_crash,
+    0 AS startup_crash,
+    0 AS content_shutdown_crash,
+    LEAST(GREATEST(payload.info.subsession_length / 3600, 0), 25) AS usage_hours,
```

Comment:
Could you add a comment here on how we're doing this to protect against clients sending absurdly high values for usage_hours? This bug has a reference: https://bugzilla.mozilla.org/show_bug.cgi?id=1514392